functions/src/routes/index.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const locationController = require('../controllers/locations');
+
+const router = express.Router();
+
+router.post('/location', locationController.create);
+router.get('/locations', locationController.getAll);
+router.get('/locations/:id', locationController.getOne);
+router.put('/locations/:id', locationController.updateLocation);
+router.delete('/locations/:id', locationController.deleteLocation);
```

Comment:
Rename page to location.js not index.js.
Remove all locations from the route name, say '/location' should be '/ '.
Go to the main index.js, import the route/location.js as location, then  under the app.use user route use this instead  app.use('/v1/location', location);

Reply:
const routes = require('./src/routes/index'); change to
const location = require('./src/routes/location');

Reply:
Nice job bro. Kudos SIr

Reply:
The `index.js` is well all the top level routers should live. create a `routes/location.js` and definedall the location specific routes there